packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/EditView/components/Events/index.js

```diff
@@ -0,0 +1,316 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+import { useFormikContext } from 'formik';
+import { useIntl } from 'react-intl';
+import styled from 'styled-components';
+import {
+  FieldLabel,
+  Flex,
+  Typography,
+  BaseCheckbox,
+  Checkbox,
+  Loader,
+  RawTable as Table,
+  RawTd as Td,
+  RawTr as Tr,
+  RawThead as Thead,
+  RawTbody as Tbody,
+} from '@strapi/design-system';
+
+import { useContentTypes } from '../../../../../../../hooks/useContentTypes';
+
+export const formatValue = (value) =>
+  value.reduce((acc, curr) => {
+    const key = curr.split('.')[0];
+
+    if (!acc[key]) {
+      acc[key] = [];
+    }
+    acc[key].push(curr);
+
+    return acc;
+  }, {});
+
+const StyledTable = styled(Table)`
+  tbody tr:nth-child(odd) {
+    background: ${({ theme }) => theme.colors.neutral100};
+  }
+
+  thead td span {
+    color: ${({ theme }) => theme.colors.neutral500};
+  }
+
+  td {
+    width: 10%;
+    vertical-align: middle;
+    text-align: center;
+  }
+
+  tr {
+    height: 52px;
+  }
```

Comment:
Why do you need that?

Reply:
Without the table rows appear too short, like this:

<img width="880" alt="image" src="https://github.com/strapi/strapi/assets/48524071/72cc3301-51a5-48f2-ae75-0a5aa452e254">

Is there a better way to make this match the designs?

Reply:
Without the table rows appear too short, like this:

<img width="880" alt="image" src="https://github.com/strapi/strapi/assets/48524071/72cc3301-51a5-48f2-ae75-0a5aa452e254">

Is there a better way to make this match the designs?

Reply:
What do you think about adding a padding to the `td`s and `th`s? I think that us much better than fixed heights.